src/proxy.c

```diff
@@ -1274,6 +1274,40 @@ cc_proxy_hyper_pod_create (struct cc_oci_config *config)
 }
 
 /**
+ * Construct an hyperstart fsmap structure
+ *
+ * \param config \ref cc_oci_config.
+ *
+ * \return \c JsonArray on success, else \c NULL.
+ */
+JsonArray *
+cc_get_hyper_fsmap(struct cc_oci_config *config)
+{
+	JsonArray  *fsmap_arr;
+	JsonObject *fsmap_desc;
+	GSList     *l;
+
+	if (! config) {
+		return NULL;
+	}
+
+	fsmap_arr = json_array_new();
+	for (l = config->oci.mounts; l && l->data; l = g_slist_next (l)) {
```

Comment:
Just the volumes. The oci.mounts should contain only volumes right? config->rootfs_mounts should contain just the rootfs mounts if I have understood correctly.